xml/config/apps/demo/Gui3LeftTree.xml

```diff
@@ -8,18 +8,22 @@
 >
 
   <name>My Layout</name>
+    <name xml:lang="cs">Moje rozvržení</name>
```

Comment:
I think this name ("My Layout") is not translatable, since it is not within a node object.